roles/static_inventory/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Either to checkpoint the dynamic inventory into a static one
+refresh_inventory: True
+inventory: static
+inventory_path: ~/openstack-inventory
```

Comment:
Everything below this line should IMO be in a separate PR that adds the bastion node.